lib/pprof/pprof.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pprof
+
+import (
+	"fmt"
+	"net/http"
+	_ "net/http/pprof"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+type PprofPort int
+
+const basePort = 6060
+
+const (
+	VchInitPort PprofPort = iota
+	ImagecPort
+	VicadminPort
+	DockerPort
+	PortlayerPort
+	maxPort
+)
+
+func StartPprof(name string, component PprofPort) error {
+	port := component + basePort
+	log.Info(fmt.Sprintf("Launching %s server on port %d", name, port))
+	go func() {
+		log.Info(http.ListenAndServe(fmt.Sprintf("127.0.0.1:%d", port), nil))
```

Comment:
It's only listening on `lo`?  Is that intentional?  How do we get to this from outside the vch?